src/AddressDriver.sol

```diff
@@ -94,6 +94,27 @@ contract AddressDriver is Upgradeable, ERC2771Context {
     /// Positive to add funds to the drips balance, negative to remove them.
     /// @param newReceivers The list of the drips receivers of the sender to be set.
     /// Must be sorted by the receivers' addresses, deduplicated and without 0 amtPerSecs.
+    /// @param maxEndTip1 The first tip to calculate the maximum end time of drips
```

Comment:
```suggestion
    /// @param maxEndTip1 optional parameter for gas optimization pass zero if it should not be used
```
I would start with a one-liner for readers who are not interested in using the parameter and are not interested in what it is exactly.  I just want them to read the first line and get all the information they need.

Reply:
Good point, added.